src/UI/button/Button.jsx

```diff
@@ -0,0 +1,23 @@
+import PropTypes from "prop-types";
+import "./Button.scss";
+
+function Button({ type, className, children, ...props }) {
```

Comment:
@Ayush-Bharadva
This is not a useful component, in my opinion. Direct use of the button is available.